src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -9229,8 +9233,16 @@ public final class UpdateAddressHttpRequest implements ApiMessage {
       return fieldMask;
     }
 
-    public Builder setFieldMask(List<String> fieldMask) {
-      this.fieldMask = fieldMask;
+    public Builder addAllFieldMask(List<String> fieldMask) {
+      if (this.fieldMask == null) {
+        this.fieldMask = new ArrayList<>(fieldMask.size());
+      }
+      this.fieldMask.addAll(fieldMask);
+      return this;
+    }
+
+    public Builder addFieldMask(String fieldMask) {
+      this.fieldMask.add(fieldMask);
```

Comment:
We also need to check for fieldMask being null here right?

Reply:
Ah sorry, I meant the `this.fieldMask`, because `addAllFieldMask` checks for it. I think they should both check or both not check.

Reply:
Ah gotcha gotcha :)
Done.